front/src/styles/scss/common/map/_map.scss

```diff
@@ -7,31 +7,66 @@
   position: absolute;
   z-index: 2;
   right: 25px;
-  top: 5rem;
+  top: 80px;
   button {
     display: block;
-    font-size: 1.5rem;
-    margin-bottom: 0.5rem;
+    font-size: 24px;
+    margin-bottom: 8px;
   }
   &.editor {
-    top: 1rem;
+    top: 16px;
+  }
+}
+
+.osrd-btn-map-container {
+  position: absolute;
+  z-index: 2;
+  top: 0;
+  right: 0;
+  width: 72px;
+  height: 360px;
+  padding: 12px;
+}
+
+.osrd-btn-map {
+  color: var(--black100);
+  width: 48px;
```

Comment:
where does 48px come from?
![Screenshot_20241121_165841](https://github.com/user-attachments/assets/a9093023-3c52-4fa7-968d-acdb6c65bffa)

Reply:
thanks. perhaps link it in the description so we know what you used as reference

Reply:
Sure